spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :request do
+  describe 'Test posts controller' do
+    it 'Should Return a successfull response' do
+      get '/users/:id/posts/'
+      expect(response).to be_successful
+    end
+
+    it 'Should render correct template(view)' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'Should render show view for user post' do
+      get '/users/:id/posts/:id'
+      expect(response).to render_template(:show)
+    end
+
+    it 'Should return the correct placeholder for post' do
+      get '/users/:id/posts'
+      expect(response.body).to include('Users Post')
+    end
+  end
```

Comment:
- [ ] Kindly make sure to test all controllers [these tests](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for,includes%20correct%20placeholder%20text.) individually. In other words, each controller has to have 3 tests which sum up to a total of 12 tests (2 controllers with 2 actions each), as per the requirement.